### PR TITLE
To `polkadot-stable2407`

### PR DESCRIPTION
**Pull request by AurevoirXavier**

### CODE REVIEW

File: node/src/cli.rs [hunk 1]
```diff
@@ -16,15 +16,8 @@
...
 
-/// The version of the node.
-///
-/// This is the version that is used for versioning this node binary.
-/// By default the `minor` version is bumped in every release. `Major` or `patch` releases are only
-/// expected in very rare cases.
-///
-/// The worker binaries associated to the node binary should ensure that they are using the same
-/// version as the main node that started them.
-pub const NODE_VERSION: &str = "6.8.1";
```
Comment (boundless-forest) [inline node/src/cli.rs:27]: If this field is deleted, how can we check the node version?

Comment (AurevoirXavier) [reply node/src/cli.rs:27]: --version

**Review by jiguantong — DISMISSED** [review]

**Comment by jiguantong** [comment]
XCM related changes LGTM

File: runtime/darwinia/src/pallets/polkadot_xcm.rs [hunk 2]
```diff
@@ -185,21 +206,17 @@ impl xcm_executor::Config for XcmExecutorConfig {
...
-			Balances,
-			DealWithFees<Runtime>,
-		>,
-		xcm_primitives::FirstAssetTrader<AssetType, AssetManager, XcmFeesToAccount>,
-	);
+	// As trader we use the XcmWeightTrader pallet.
+	// For each foreign asset, the fee is computed based on its relative price (also
+	// stored in the XcmWeightTrader pallet) against the native asset.
+	// For the native asset fee is computed using WeightToFee implementation.
+	type Trader = pallet_xcm_weight_trader::Trader<Runtime>;
```
Comment (aurexav) [inline runtime/darwinia/src/pallets/polkadot_xcm.rs:213]: Add migration.

**Pull request merged**